File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.cc
#include <stdio.h>

int main(void) {
	int arr[30] = { 0 }, n;
	for (int i = 0; i < 28; i++) {
		scanf("%d", &n);
		arr[n - 1]++;
	}
	for (int i = 0; i < 30; i++) {
		if (arr[i] == 0)
			printf("%d\n", i + 1);
	}
	return 0;
}

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int[] arr = new int[30];
        int a;
        for(int i = 0; i < 28; i++) {
            a = sc.nextInt();
            arr[a - 1]++;
        }
        for(int i = 0; i < 30; i++) {
            if(arr[i] == 0)
                System.out.println(i + 1);
        }
    }
}